src/resources/postcss/tickets-admin/settings/_tickets-commerce.pcss

```diff
@@ -463,3 +463,100 @@
 		color: #405bfb;
 	}
 }
+
+.tec-tickets__admin-settings-tickets-commerce-section-menu {
+	padding-top: var(--tec-spacer-6);
+}
+
+.tec-tickets__admin-settings-tickets-commerce-section-menu-link {
+	border-bottom-color: #c4c4c4;
+	border-bottom-style: solid;
+	border-bottom-width: 1px;
```

Comment:
Do we have a var for this in common styles?

Reply:
No. I checked both the colors and border vars. None exist for those values.